src/development/environment-variables.md

```diff
@@ -52,12 +73,7 @@ the runtime (*ie. PHP*) and prefixed with `PLATFORM_*`.
 * **PLATFORM_VARIABLES**: A base64-encoded JSON object which keys are variables names and values are variable values (*a string*).
 * **PLATFORM_PROJECT_ENTROPY**: A random value created when the project is first created, which is then stable throughout the project’s life. This can be used for Drupal hash salt, Symfony secret, or other similar values in other frameworks.
 
-Since values can change over time, the best thing is to just introspect the variable at runtime and use it to configure your application.
-
-For example with Drupal, we use the **PLATFORM_RELATIONSHIPS** variable
-to configure your `settings.local.php`.
-
-For example:
+Since values can change over time, the best thing is to just introspect the variable at runtime and use it to configure your application. For example:
```

Comment:
"introspect" doesn't make sense here - let's fix that at the same time, perhaps just:

> the best thing is to use the variable at runtime to configure your application